plugins/klaxon-jackson/src/main/kotlin/com/beust/klaxon/jackson/JacksonParser.kt

```diff
@@ -118,7 +117,8 @@ private fun KlaxonJson.parseValue(node: JsonNode): Any {
             }
         }
         JsonNodeType.BINARY -> Base64.getEncoder().encodeToString(node.binaryValue())
-        JsonNodeType.NULL, JsonNodeType.MISSING -> throw KlaxonException("Unexpected node type: $nodeType")
+        JsonNodeType.NULL -> null
```

Comment:
@cbeust This was the actual problem right here. Improperly handling `JsonNodeType.NULL`.